seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/excel/local_excel_projection_to_assert.conf

```diff
@@ -18,16 +18,41 @@
 ###### This config file is a demonstration of streaming processing in seatunnel config
 ######
 
+env {
```

Comment:
It's duplicate with #4615 https://github.com/apache/incubator-seatunnel/pull/4615/files#diff-ee22b91619cbf4eea14ae402900bf2bb469e4ad4a2a6d1387af1c3db93d5d7fcR21